src/main/java/duke/command/ShowCommand.java

```diff
@@ -38,6 +38,9 @@ public void execute(TaskList taskList, Ui ui, Storage storage) {
         case "done":
             taskList.markTaskAsDone(replyArr);
             break;
+        case "find":
```

Comment:
Instead of your `showCommand.java` class handling this input, why not let your `Parser.java` class call all these tasks for you in its current switch case? Right now, it looks like you have to handle the logic twice, which might get messier as we have to go along and implement more features (or expand other ones)!